ci/flaky_test/process_xml.py

```diff
@@ -100,22 +100,23 @@ def getGitInfo(CI_TARGET):
     sys.exit(0)
   output_msg += getGitInfo(CI_TARGET)
 
+  find_dir = '${TEST_TMPDIR}/**/**/**/**/bazel-testlogs/'
   if CI_TARGET == "MacOS":
-    os.system('find ${TEST_TMPDIR}/ -name "attempt_*.xml" > ${TMP_OUTPUT_PROCESS_XML}')
-  else:
-    os.system(
-        'find ${TEST_TMPDIR}/**/**/**/**/bazel-testlogs/ -name "attempt_*.xml" > ${TMP_OUTPUT_PROCESS_XML}'
-    )
+    find_dir = '${TEST_TMPDIR}/'
+  os.system(
+      'sh -c "/usr/bin/find {} -name attempt_*.xml > ${{TMP_OUTPUT_PROCESS_XML}}"'.format(find_dir))
 
   f = open(os.environ['TMP_OUTPUT_PROCESS_XML'], 'r+')
```

Comment:
nit: `with open('foo.txt', "r") as file:` would be more pythonic and it would take care of the close file in most cases.

Reply:
good call! totally forgot about the `with` syntax